docs/examples/classification/medmnist_organ.md

```
@@ -0,0 +1,508 @@
+```
+
+```python
+x, y = next(iter(train_ds))
+```
+```python
+plot_sample(
+    x, y, sample_idx=0, max_slices=4
+)
+```
+
+![](../../assets/examples/medmnist_orgran3d/organ3d_sample1.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)